src/app/clusters/ota-requestor/DefaultOTARequestorDriver.h

```diff
@@ -84,10 +84,10 @@ class DefaultOTARequestorDriver : public OTARequestorDriver
 protected:
     void StartPeriodicQueryTimer();
     void StopPeriodicQueryTimer();
-    void PeriodicQueryTimerHandler(System::Layer * systemLayer, void * appState);
+    static void PeriodicQueryTimerHandler(System::Layer * systemLayer, void * appState);
```

Comment:
nit: Maybe group the static functions at the top to separate from non-static functions for easier reading.

Reply:
Done.